src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -115,5 +130,26 @@ public void SuppressOutput()
             PostContent.Clear();
             PostElement.Clear();
         }
+
+        /// <summary>
+        /// A delegate used to execute and retrieve the rendered child content asynchronously.
```

Comment:
"execute _children_" might make this read a bit better. Odd to think of executing child content. Applies in the associated constructor `<param>` and the other overload.